prboom2/src/dsda/dgeom.h

```diff
@@ -0,0 +1,212 @@
+//
+// Copyright(C) 2023 Brian Koropoff
+//
+// This program is free software; you can redistribute it and/or
+// modify it under the terms of the GNU General Public License
+// as published by the Free Software Foundation; either version 2
+// of the License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// DESCRIPTION:
+//	Internal double-precision geometry functions
+//
+//      Not intended for use outside of DSDA subdir, so these aren't
+//      prefixed with dsda_
+//
+
+#ifndef DSDA_INTERNAL_
+#error Internal header, not for use outside dsda subdir
```

Comment:
This protection doesn't look particularly necessary to me. I do think we need a prefix though, `dgeom_` maybe. Unprefixed simple names like `Len` don't capture their specificity otherwise.

Reply:
I was trying to avoid renaming everything, but I can add a prefix.